src/ngraph/op/lrn.cpp

```diff
@@ -34,7 +45,9 @@ op::LRN::LRN(const Output<Node>& arg, double alpha, double beta, double bias, si
 
 void op::LRN::validate_and_infer_types()
```

Comment:
If `axes` is a constant and `arg`'s rank is static, we need a check to make sure that all axes are in bounds. We'll need unit tests for that check, as well.

Reply:
- Checking shape of `axes` was added
- Unit tests to `arg` and `axes` was added